Trunk.toml

```diff
@@ -8,6 +8,8 @@ trunk-version = "*"
 target = "index.html"
 # Build in release mode.
 release = false
+# Use a custom cargo profile
```

Comment:
```suggestion
# Use a custom cargo profile. This will override the profile chosen by `release = {true|false}`.
```

Reply:
Good point. I am not happy with the message though. I'll try to make it a bit more specific. If I fail, I'll revert to yours :grin: